tests/testthat/test-utilities-units.R

```diff
@@ -171,20 +171,20 @@ test_that("It returns NULL when the dimension exists,
 
 context("hasUnit")
 test_that("It returns true for an existing unit in the dimension, false otherwise", {
-  expect_true(hasUnit(unit = "µmol", dimension = "Amount"))
+  expect_true(hasUnit(unit = encodeUnit("µmol", dimension = "Amount")))
```

Comment:
wrong bracketness, should be `expect_true(hasUnit(unit = encodeUnit("µmol"), dimension = "Amount"))`

Reply:
in fact here encodeUnit is not required, because it's done by the function hasUnit:
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/fac585f3fc4874bbdab83f040fa76d96df99b487/R/utilities-units.R#L30-L35